mltrace/client.py

```diff
@@ -443,6 +443,12 @@ def get_recent_run_ids(limit: int = 5, last_run_id=None):
     return store.get_recent_run_ids(limit, last_run_id)
 
 
+def get_all_run_ids(last_run_id=None):
```

Comment:
This will only get the last 50 run ids by default. Also no need for this function

Reply:
removed this and implemented the cli feature differently by directly requiring the db